incubator/kafka/values.yaml

```diff
@@ -143,52 +143,86 @@ persistence:
   ##
   # storageClass:
 
-## Metrics
+## Port the jmx metrics are exposed in native jmx format, not in Prometheus format
+jmxPort: 5555
+
+## Prometheus Exporters / Metrics
 ##
-metrics:
+prometheus:
```

Comment:
can you tell us a little more about this change and why there are two ports?

Reply:
Sure, I renamed to try and make the distinction b/t the 2 ports a little more clear.  There is a top level `jmxPort` var now which is the actually jmx port that exposes metrics in jmx format.  The JMX exporter for prometheus transforms those stats into a prometheus format and exposes them for scraping at `prometheus.jmx.port`

I had a couple question on it, so I thought renaming `metrics` to `prometheus` made things more clear.  If someone has a better suggestion let me know, but I think its clearer since metrics could be jmx or prometheus style.